website/src/app/kb/client-apps/windows-gui-client/readme.mdx

```diff
@@ -77,6 +77,39 @@ Firezone.
 When you're signed out, you can't access private Resources, and the computer
 will use its normal DNS and Internet behavior.
 
+## MDM configuration
```

Comment:
This should be moved to the `Provision via MDM` section in `/kb/deploy/clients` since the intention behind this page was more for end-users.